model-optimizer/extensions/ops/scatter.py

```diff
@@ -94,6 +100,32 @@ class ScatterElementsUpdate(Scatter):
     op = op_type = 'ScatterElementsUpdate'
     version = 'opset3'
 
+    @staticmethod
+    def infer(node: Node):
+        Scatter.infer(node)
+
+        node_name = node.soft_get('name', node.id)
+        connected_in_ports = [port for port in node.in_ports().values() if not port.disconnected()]
+        assert len(connected_in_ports) == 4, \
+            "Incorrect number of inputs for {} node".format(node_name)
+
+        input_value = node.in_port(0).data.get_value()
+        indices_value = node.in_port(1).data.get_value()
+        indices_shape = node.in_port(1).data.get_shape()
+        updates_value = node.in_port(2).data.get_value()
+        axis = node.in_port(3).data.get_value()
+        if axis.size == 1:
```

Comment:
What if the size is not 1? We don't support such a case, so this should be an assert in this case